Library/Homebrew/.rubocop.yml

```diff
@@ -26,9 +25,26 @@ Lint/NestedMethodDefinition:
 Lint/ParenthesesAsGroupedExpression:
   Enabled: true
 
+Metrics/BlockNesting:
+  Max: 5
+
+Metrics/ModuleLength:
+  Max: 360
```

Comment:
Good idea to just setting these to high values. Mind doing that for line length, too? Would be good to get it down eventually but at least stop it growing for now.